src/index.js

```diff
@@ -320,13 +320,22 @@ export function app(state, actions, view, container) {
 
       while (k < children.length) {
         var oldKey = getKey(oldChildren[i])
+        var nextOldKey = getKey(oldChildren[i + 1])
```

Comment:
@SkaterDad Can we inline this call with the `if (newKey != null && newKey === nextOldKey) {` later below?

Reply:
You bet.  Looks like that shaves ~3 bytes off after gzip.

Reply:
Done :+1: